content/node-runners/setup/docker.md

```diff
@@ -114,38 +62,23 @@ When the installation finishes, Docker starts automatically. The "whale" icon in
 #### For WINDOWS users: Open a command-line terminal and type the following command:
 
 ```bash
-docker run --cap-add NET_ADMIN -p 4449:4449 --name myst -v $YOUR_MYSTERIUM_DIR:/var/lib/mysterium-node mysteriumnetwork/myst:latest service --agreed-terms-and-conditions wireguard
+docker run --cap-add NET_ADMIN -d -p 4449:4449 --name myst -v myst-data:/var/lib/mysterium-node mysteriumnetwork/myst:latest service --agreed-terms-and-conditions
 ```
 
-**_Note 1:_** Replace `$YOUR_MYSTERIUM_DIR` with the path where you'd like to store the node's configuration and keystore files, e.g.
-
-```bash
-export YOUR_MYSTERIUM_DIR=~/.mysterium
-```
+**_Note 1:_** Replace `myst-data` with the path where you'd like to store the node's configuration and keystore files, e.g.
 
 **_Note2:_** By adding `--agreed-terms-and-conditions` command line option you accept our Terms & Conditions.
 
-**_Note3:_** You can use Docker detached mode by adding the option `--detach` or `-d`.
-
-
 #### For MacOS users: Open a command-line terminal and type the following command:
 
 ```bash
-docker run --cap-add NET_ADMIN -p 4449:4449 --name myst -v $YOUR_MYSTERIUM_DIR:/var/lib/mysterium-node --device /dev/net/tun:/dev/net/tun mysteriumnetwork/myst:latest service --agreed-terms-and-conditions wireguard
+docker run --cap-add NET_ADMIN -d -p 4449:4449 --name myst -v myst-data:/var/lib/mysterium-node --device /dev/net/tun:/dev/net/tun mysteriumnetwork/myst:latest service --agreed-terms-and-conditions
 ```
 
-**_Note 1:_** Replace `$YOUR_MYSTERIUM_DIR` with the path where you'd like to store the node's configuration and keystore files, e.g.
-
-```bash
-export YOUR_MYSTERIUM_DIR=~/.mysterium
-```
+**_Note 1:_** Replace `myst-data` with the path where you'd like to store the node's configuration and keystore files, e.g.
 
 **_Note2:_** By adding `--agreed-terms-and-conditions` command line option you accept our Terms & Conditions.
 
-**_Note3:_** You can use Docker detached mode by adding the option `--detach` or `-d`.
-
-
-
 ## Complete installation
 
-Once the container is running please [log into the Node UI](/node-runners/node-ui/) to set up your service pricing, payout address and claim your node in [MMN](https://testnet2.mysterium.network) to receive bounties.
+Once the container is running please [log into the Node UI](/node-runners/node-ui/) to set up your service pricing, payout address and claim your node in [MMN](https://my.mysterium.network) to receive bounties.
```

Comment:
Let's keep `https://my.mysterium.network` domain instead of `testnet2` `testnet3`. No need to update it for every release.